### PR TITLE
Update eclipse-packaging.jsonnet

### PR DESCRIPTION
**Pull request by merks**
The name is Eclipse Packaging Project not Eclipse Packaging Product

### CODE REVIEW

**Comment by merks** [comment]
@jonahgraham   

Fixing a tiny typo.

@netomi 

Maybe fixing a tiny typo doesn't need to wait for project lead approval given this is definitely the project name:

https://projects.eclipse.org/projects/technology.packaging

There's also no rush!

**Comment by netomi** [comment]
Indeed, we have plans to further work on auto approval / merge rules to simplify such changes, for now I can just approve.

**Review by netomi — APPROVED** [review]

**Pull request merged**
